Source/WTF/wtf/Assertions.cpp

```diff
@@ -80,7 +80,7 @@ static String createWithFormatAndArguments(const char* format, va_list args)
 ALLOW_NONLITERAL_FORMAT_BEGIN
 
 #if USE(CF)
-    if (strstr(format, "%@")) {
+    if (memmemSpan(span8(format), "%@"_span) != notFound) {
```

Comment:
I think that Geoff's right and the `span8` function that takes `char*` should be renamed to `unsafeSpan8`.

I also find the `memmem` name so annoying I might even suggest we rename `memmemSpan` to `find`. After all, part of the style of `memmem` is that it returns a pointer and we don't do that. Also have one that returns a boolean called `contains` to avoid all the `notFound` stuff.

Reply:
Ok, I can rename the `span()` / `span8()` functions in a follow-up since they are not new in this PR.

Reply:
And yes, I'll happily rename `memmemSpan()` to `find()`. I had the same thought when I wrote this patch. I was initially planning to do in a follow-up but I guess I'll rename in this PR.

Reply:
I also added `contains()`